src/libutil/tests/meson.build

```diff
@@ -0,0 +1,53 @@
+# libutil.tests build file
+#============================================================================
+
+libutil_tests_inc = [include_directories('.')]
+
+# dependancies
+#============================================================================
+
+libutil_tests_dep_list = [
+    gtest_dep,
+
+    libutil_dep]
+
+
+# src files
+#============================================================================
+
+libutil_tests_src = files(
+    'closure.cc',
+    'compression.cc',
+    'config.cc',
+    'hash.cc',
+    'json.cc',
+    'logging.cc',
+    'lru-cache.cc',
+    'main.cc',
```

Comment:
There is no `src/libutil/tests/main.cc`.

Reply:
file seems to not have been added back. its here:

https://github.com/p01arst0rm/nix/blob/meson-rebase/src/libutil/tests/main.cc

Reply:
added back now, should be good.